src/site/content/en/blog/sanitizer/index.md

```diff
@@ -0,0 +1,380 @@
+
+**`dropElements`**: Names of elements the sanitizer should remove, along with their children.
+
+
+```js
+const str = `hello <b><i>world</i></b>`
+
+new Sanitizer().sanitizeFor("div", str)
+// <div>hello <b><i>world</i></b></div>
+
+new Sanitizer({allowElements: [ "b" ]}).sanitizeFor("div", str)
+// <div>hello <b>world</b></div>
+
+new Sanitizer({blockElements: [ "b" ]}).sanitizeFor("div", str)
+// <div>hello <i>world</i></div>
+
+new Sanitizer({allowElements: []}).sanitizeFor("div", str)
+// <div>hello world</div>
+```
+
+You can also control how the sanitizer treats the specified attributes with the following options:
+
+**`allowAttributes`**: Attribute Match List to allow
+
+**`dropAttributes`**: Attribute Match List to drop
+
+The attribute properties expect **Attribute Match Lists** - objects whose keys are attribute names, and values are lists of target elements or the "**`*`**" wildcard.
+
+
+```js
+const str = `<span id=foo class=bar style="color: red">hello</span>`
+
+new Sanitizer().sanitizeFor("div", str)
+// <div><span id="foo" class="bar" style="color: red">hello</span></div>
+
+new Sanitizer({allowAttributes: {"style": ["span"]}}).sanitizeFor("div", str)
+// <div><span style="color: red">hello</span></div>
+
+new Sanitizer({allowAttributes: {"style": ["p"]}}).sanitizeFor("div", str)
+// <div><span>hello</span></div>
+
+new Sanitizer({allowAttributes: {"style": ["*"]}}).sanitizeFor("div", str)
+// <div><span style="color: red">hello</span></div>
+
+new Sanitizer({dropAttributes: {"id": ["span"]}}).sanitizeFor("div", str)
+// <div><span class="bar" style="color: red">hello</span></div>
+
+new Sanitizer({allowAttributes: {}}).sanitizeFor("div", str)
+// <div>hello</div>
+```
+
+**`allowCustomElements`** is the option to allow or deny custom elements. If they're allowed, other configurations for elements and attributes still apply.
+
+
+```js
+const str = `<custom-elem>hello</custom-elem>`
+
+new Sanitizer().sanitizeFor("div", str);
+// <div></div>
+
+new Sanitizer({ allowCustomElements: true,
+                allowElements: ["div", "custom-elem"]
+              }).sanitizeFor("div", str);
+// <div><custom-elem>hello</custom-elem></div>
+```
+
+{% Aside %}
+The Sanitizer API is designed to be safe by default. This means that no matter how you set it up, it will never allow constructs that are known XXS targets. For example, `allowElements: ["script"]` won't actually allow `<script>`, because the built-in baseline configuration cannot be overridden. The purpose of customization is to override default settings if your application has special needs.
+{% endAside %}
+
+
+## API Surface
+
+It may be illustrative to compare the Sanitizer API to [DOMPurify](https://github.com/cure53/DOMPurify), a well-known library that offers sanitization functionality: The main difference is that DOMPurify may returns the result of the sanitization as a string, which you need to expand into a DOM via `.innerHTML`.
```

Comment:
```suggestion
It may be illustrative to compare the Sanitizer API to [DOMPurify](https://github.com/cure53/DOMPurify), a well-known library that offers sanitization functionality. The main difference is that DOMPurify may returns the result of the sanitization as a string, which you need to expand into a DOM via `.innerHTML`.
```